support-frontend/assets/pages/weekly-subscription-checkout/components/thankYou.tsx

```diff
@@ -40,6 +46,14 @@ type PropTypes = ConnectedProps<typeof connector> & {
 	orderIsGift: boolean;
 };
 
+const checkmarkListStyles = css`
+	width: 100%;
```

Comment:
Is this `100%` needed? Or is that the `initial` value?

Reply:
Good spot, removed and tested, thanks.